TSLA-4256 Docs: managed cluster versions (#891)
tigera/docs: calico-cloud/get-started/upgrade-cluster.mdx

To upgrade managed clusters to the latest version of {{prodname}}:

1. In Manager UI, go to **Managed Clusters**.

> 1. Click the **Actions** link on the cluster you want to reinstall, and then select  **ReInstall**.

Reviewer: ```suggestion
1. For the cluster you want to upgrade, select **Actions** > **Reinstall**.
```

Author: done

1. Select the version of {{prodname}} you want to install, you can select the currently installed version, or any newer supported version
   :::note

   Currently the only option available to install is the latest version. More versions will become available as newer versions are released
   and as older versions become qualified.

   :::

> 1. Click **Next**, then copy the command and apply it to appropriate cluster.

Reviewer: ```suggestion
1. Click **Next**, copy the command, and apply it to appropriate cluster.
```

Author: done
